src/screens/Home/components/HomeMenuItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import CheckIcon from '@material-ui/icons/Check';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+type sublabel = {
+  label: string;
+  amount: number;
+};
+
+interface HomeMenuItemProps {
+  label: string;
+  amount?: number;
+  sublabels?: sublabel[];
+  noBadge?: boolean;
+  classes: { root: string; content: string; noAlert: string; sublabel: string; icon: string; grey: string };
+}
```

Comment:
Naming aside, this entire component is really clean and easy to read. It sparks joy. :) thank u.